recipes/openssh/openssh.inc

```diff
@@ -72,13 +72,20 @@ inherit auto-package-utils
 AUTO_PACKAGE_UTILS = "ssh-add ssh-agent ssh-keyscan ssh-keygen ssh-keysign \
 	scp ssh sshd sftp sftp-server ssh-pkcs11-helper"
 
+PACKAGES =+ "${PN}-ssh-config ${PN}-sshd-config"
+
 ALLOW_EMPTY_${PN} = "1"
-FILES_${PN}-ssh = "${sysconfdir}/ssh/ssh_config ${bindir}/slogin"
+FILES_${PN}-ssh = "${bindir}/slogin"
 FILES_${PN}-ssh-doc = "${mandir}/*/ssh_config.* ${mandir}/*/slogin.*"
 FILES_${PN}-sshd = "${sysconfdir}/init.d/sshd \
-	${sysconfdir}/ssh/moduli ${sysconfdir}/ssh/sshd_config \
+	${sysconfdir}/rcS.d/* \
+	${sysconfdir}/ssh/moduli \
 	${localstatedir}/run/sshd"
 FILES_${PN}-sshd-doc = "${mandir}/*/moduli.* ${mandir}/*/sshd_config.*"
+FILES_${PN}-ssh-config = "${sysconfdir}/ssh/ssh_config"
+PROVIDES_${PN}-ssh-config = "ssh-config"
+FILES_${PN}-sshd-config = "${sysconfdir}/ssh/sshd_config"
+PROVIDES_${PN}-sshd-config = "sshd-config"
```

Comment:
No, they are actually both not really needed. You can just provide openssh-ssh-config and/or openssh-sshd-config, which is probably better, as such packages are very unlikely to work with other ssh packages than openssh.